source/fundamentals/enterprise-auth.txt

```diff
@@ -60,6 +60,17 @@ Select the :guilabel:`Connection String` or the :guilabel:`MongoCredential`
 tab below for instructions and sample code for specifying this authentication
 mechanism:
 
+.. note::
+   The Java Driver will by default cache 
+   `Kerberos tickets <https://docs.oracle.com/javase/7/docs/api/javax/security/auth/kerberos/KerberosTicket.html>`__
+   by ``MongoClient`` instance. If your deployment needs to frequently create and destroy
+   ``MongoClient`` instances, you can change the default Kerberos ticket caching behavior of the Java Driver
+   to cache by process. To cache Kerberos tickets by process, you must specify the ``JAVA_SUBJECT_PROVIDER``
+   mechanism property and provide a 
+   `KerberosSubjectProvider <https://mongodb.github.io/mongo-java-driver/4.2//apidocs/mongodb-driver-core/com/mongodb/KerberosSubjectProvider.html#%3Cinit%3E()>`__
+   in your ``MongoCredential`` instance.
+
+
```

Comment:
nitpick: placement

Reading this, the note feels out of place on the page in its current location. 

suggestion: 

Perhaps moving it further down after the code block, removing the note, and using a subheading might clean up the presentation and make it more discoverable?

Reply:
Thank you for the suggestions @terakilobyte. I agree that the note felt a bit out of place. I moved the note's content to the bottom of the connection string/MongoCredential tab section of the page. Moving the content also provided an opportunity to add a code snippet in the MongoCredential tab showing how to perform the Kerberos ticket caching configuration, so I added that as well. Let me know if you are happy with this solution.

Staging link: https://docs-mongodbcom-staging.corp.mongodb.com/228a552/java/docsworker-xlarge/DOCSP-13803-kerberos-ticket-caching/fundamentals/enterprise-auth/#std-label-gssapi-auth-mechanism